libcst/metadata/tests/test_name_provider.py

```diff
@@ -287,3 +344,56 @@ class Foo:
         self.assertEqual(
             names[attribute], {QualifiedName("a.aa.aaa", QualifiedNameSource.IMPORT)}
         )
+
+
+class FullyQualifiedNameProviderTest(UnitTest):
+    def test_builtins(self) -> None:
+        qnames = get_fully_qualified_names(
+            "test/module.py",
+            """
+            int(None)
+            """,
+        )
+        qnames -= {QualifiedName(name="test.module", source=QualifiedNameSource.LOCAL)}
```

Comment:
This is not a good test because it doesn't guarantee the test.module is in qnames. This line has no effect if test.module is not in qnames.
Can we explicitly test the test.module qualified is available in qnames and it's associate with the Module node?

Same comment for `def test_imports`.